src/components/bcros/PartyInfo.vue

```diff
@@ -2,30 +2,41 @@
   <BcrosAccordion
     :name="name"
     :items="partyItems"
+    :disabled="disableExpand"
   />
 </template>
 
 <script setup lang="ts">
 import { storeToRefs } from 'pinia'
 
 const business = useBcrosBusiness()
+const bootstrap = useBcrosBusinessBootstrap()
 const { currentParties } = storeToRefs(business)
+const { bootstrapFiling } = storeToRefs(bootstrap)
 
 const props = defineProps({
   name: { type: String, required: true },
   roleType: { type: String, required: true },
   showEmail: { type: Boolean, required: true },
-  expandTopItem: { type: Boolean, default: false }
+  expandTopItem: { type: Boolean, default: false },
+  showAddress: { type: Boolean, default: true }
+})
+
+const disableExpand = computed(() => {
+  return !currentParties?.value?.parties && !!bootstrapFiling?.value?.filing?.incorporationApplication?.parties
```

Comment:
did you want to use && or maybe || when should expand be disabled ?
if no currentparties but incorporationApplication has parties set ?

Reply:
I think we want && it makes it default to current functionality